src/Chainweb/Utils.hs

```diff
@@ -1368,6 +1372,18 @@ showIpv6 ha = T.intercalate ":"
   where
     (a0,a1,a2,a3,a4,a5,a6,a7) = hostAddress6ToTuple ha
 
+-- -------------------------------------------------------------------------- --
+-- GHC Host architecture
+
+hostArch :: String
+#if aarch64_HOST_ARCH == 1
+hostArch = "aarch64"
+#elif x86_64_HOST_ARCH == 1
+hostArch = "x86_64"
+#else
+hostArch = "unknown"
+#endif
+
```

Comment:
An alternative would be to make use of the `uname` systems call at runtime. However, the resulting `utsname` structure is underspecified in POSIX (the length of the fields varies between different implementations). This would require either the implementation of c-wrappers for accessing the fields, or the use of a preprocessor (e.g. hsc) or configure script, or encoding a heuristic in C-macros similar to above code.

Overall, above macro that statically gets the value from GHC seems the simplest approach. If needed, it can be easily adjusted in the future by  adding more cases.